pages/linux/qm-sendkey.md

```diff
@@ -0,0 +1,12 @@
+# qm sendkey
+
+> Send the QEMU monitor encoding key event to a specific virtual machine.
```

Comment:
```suggestion
> Send  QEMU monitor encoding key event to a specific virtual machine.
```
I updated the suggestion seems like you committed it fast 😄

Reply:
Ah I was wondering what happened. I will wait for few more minutes before I commit 😅

Reply:
> Ah I was wondering what happened. I will wait for few more minutes before I commit sweat_smile

I guess you can commit it now.